services/fleet.js

```diff
@@ -62,13 +60,47 @@ async function getScriptByEndpoint(endpointList){
         }
     }
     return allScripts;
+}
+
+const mergeEndpointAndScripts = (endpoints, scriptsData) => {
+    // Ensure endpoints is an array
+    if (!Array.isArray(endpoints)) {
+        throw new TypeError('Expected endpoints to be an array');
+    }
+
+    console.log('Scripts Data:', scriptsData); // Debug output
```

Comment:
Is it necessary? it prints out a lot of lines and makes it hard to watch for error

Reply:
deleted